x/bank/keeper.go

```diff
@@ -108,6 +108,7 @@ func (keeper ViewKeeper) HasCoins(ctx sdk.Context, addr sdk.Address, amt sdk.Coi
 //______________________________________________________________________________________________
 
 func getCoins(ctx sdk.Context, am sdk.AccountMapper, addr sdk.Address) sdk.Coins {
+	ctx.GasMeter().ConsumeGas(10, "getCoins")
```

Comment:
let's get a list of constants for these ... possibly parameters one day ...

Reply:
Switched to constants at top of file. Also, these gas costs are probably wrong / may be unnecessary with the GasKVStore.

Reply:
https://github.com/cosmos/cosmos-sdk/issues/1008